terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf

```
@@ -14,6 +14,10 @@ data "aws_iam_policy_document" "create_a_derived_table" {
       "arn:aws:s3:::mojap-derived-tables",
       "arn:aws:s3:::dbt-query-dump/*",
       "arn:aws:s3:::dbt-query-dump",
+      "arn:aws:s3:::mojap-manage-offences/ho-offence-codes/*",
+      "arn:aws:s3:::mojap-manage-offences/ho-offence-codes",
```

Hi sorry should have been clearer - the non-starred ones only refer to the `getBucket` powers, so should stay as was.

```suggestion
      "arn:aws:s3:::mojap-manage-offences/",
```